api-report/sequence.api.md

```diff
@@ -78,10 +78,10 @@ export function createIdIntervalIndex<TInterval extends ISerializableInterval>()
 export function createOverlappingIntervalsIndex<TInterval extends ISerializableInterval>(client: Client, helpers: IIntervalHelpers<TInterval>): IOverlappingIntervalsIndex<TInterval>;
 
 // @public (undocumented)
-export function createOverlappingSequenceIntervalsIndex(client: Client): SequenceIntervalIndexes.Overlapping;
+export function createOverlappingSequenceIntervalsIndex(sharedString: SharedString): SequenceIntervalIndexes.Overlapping;
 
 // @public (undocumented)
-export function createStartpointInRangeIndex<TInterval extends ISerializableInterval>(helpers: IIntervalHelpers<TInterval>, client: Client): IStartpointInRangeIndex<TInterval>;
+export function createStartpointInRangeIndex<TInterval extends ISerializableInterval>(client: Client, helpers: IIntervalHelpers<TInterval>): IStartpointInRangeIndex<TInterval>;
```

Comment:
related to Tony's bit about interval helpers--since we're changing the API to not need them we should remove their export. Probably should deprecate here and remove in `next` to be safe.